durabletask/task.py

```diff
@@ -71,6 +71,20 @@ def is_replaying(self) -> bool:
         """
         pass
 
+    @property
+    @abstractmethod
+    def custom_status(self) -> str:
+        """Get the custom status.
```

Comment:
I don't think we need a getter for this. We treat custom status as "write-only" in other SDKs since there shouldn't generally be a need for reading the custom status in this way (you can use local variables instead if you want to remember it).

Reply:
done